SourceS/paths.cpp

```diff
@@ -60,6 +70,20 @@ const std::string &GetConfigPath()
 	return *configPath;
 }
 
+const std::string &GetTtfPath()
+{
+	if (ttfPath == NULL)

```

Comment:
Shouldn't we be using `nullptr` instead?

Reply:
Yes, but we can wait for @AJenbo to configure .clang-tidy the way he likes and then automatically migrate all of these